web-components/src/components/theme/Theme.ts

```diff
@@ -269,6 +269,9 @@ export class Theme extends LitElement {
       <div class="theme-wrapper">
         <style>
           ${styles}
+          .theme-wrapper {
+            width: 100%;
+          }
```

Comment:
Hm, what's the context here, what's this achieving? Can a style tag apply to the parent wrapper like that? Seems like there might be a better practice if a width correction is needed.

Reply:
the inner container should be stretched to the parent, but for some reason this does not happen in the storybook, which does not make it possible to stretch the component to the full width of the screen when necessary. So it's just small correction